emcc.py

```diff
@@ -918,6 +922,24 @@ def detect_fixed_language_mode(args):
       if separate_asm:
         shared.Settings.SEPARATE_ASM = os.path.basename(asm_target)
 
+      if 'EMCC_STRICT' in os.environ:
+        shared.Settings.STRICT = os.environ.get('EMCC_STRICT') != '0'
+
+      STRICT = ([None] + filter(lambda x: x.startswith('STRICT='), settings_changes))[-1]
+      if STRICT:
```

Comment:
I worry about creating a var called `STRICT`, might confuse us in the future as to what to use. How about `temp_strict` or `strict_changes`?